holmes/plugins/prompts/kubernetes_workload_ask.jinja2

```diff
@@ -53,9 +48,7 @@ In general:
 Style guide:
 * Be painfully concise.
 * Leave out "the" and filler words when possible.
-* Be terse but not at the expense of leaving out important data like the root cause and how to fix.
-* if asked by Global Instructions or instructions in a triple single quotes section to explicitly include something in the answer, don't leave it out.
-* return a json object with the following schema as a result:
+* your answer should ONLY return a json object with the following schema as a result:
 {
```

Comment:
A. We still need to add it here if we are passing it as response_format?
B. Can we load it from the models.py and use it here instead of defining it twice?

Reply:
A. yea it makes a difference according to my tests.
B. will update

Reply:
I left it as is eventually, seems to give the best result otherwise it fails